### PR TITLE
fix: improve project setup

### PR DESCRIPTION
**Pull request by rezaerami**
## Reason: improve project setup

Closes [issue link]

## Changes made (preferably with images/screenshots):
- remove duplicated git ignore paths
- remove duplicated ts configs
- add readme
- enabled rush check

## Check off the following:

- [ ] I have reviewed my changes and run the appropriate tests.
- [ ] I have have run `rush change` to add the appropriate change logs.
- [ ] I have added/edited docs.
- [ ] I have added tutorials.
- [ ] I have double checked and DEFINITELY added docs.

### CODE REVIEW

File: packages/apps/transfer/.gitignore [hunk 1]
```diff
@@ -1,29 +1,17 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (rezaerami) [inline packages/apps/transfer/.gitignore:1]: resolves: https://github.com/kadena-community/kadena.js/pull/376#discussion_r1224242626

File: packages/apps/transfer/README.md [hunk 2]
```diff
@@ -1,38 +1,99 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (rezaerami) [inline packages/apps/transfer/README.md:1]: resolves https://github.com/kadena-community/kadena.js/pull/376#discussion_r1224243552

File: packages/apps/transfer/tsconfig.json [hunk 3]
```diff
@@ -6,15 +6,12 @@
     "lib": ["dom", "dom.iterable", "esnext"],
```
Comment (rezaerami) [inline packages/apps/transfer/tsconfig.json:6]: resolves https://github.com/kadena-community/kadena.js/pull/376#discussion_r1224264658

File: rush.json [hunk 4]
```diff
@@ -428,7 +428,7 @@
       "packageName": "@kadena/transfer",
```
Comment (rezaerami) [inline rush.json:428]: resolves https://github.com/kadena-community/kadena.js/pull/376#discussion_r1224265290

**Review by MRVDH — APPROVED** [review]

**Review by KristinaSpasevska — APPROVED** [review]

**Pull request merged**